index.md

```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
 questions and work closely with Earth system and ecosystem modelers to
 integrate the human decision components of GCAM into their analyses.
 
-## Model Overview
-
-GCAM is a dynamic-recursive model with technology-rich representations
-of the economy, energy sector, land use and water linked to a climate
-model that can be used to explore climate change mitigation policies
-including carbon taxes, carbon trading, regulations and accelerated
-deployment of energy technology. Regional population and labor
-productivity growth assumptions drive the energy and land-use systems
-employing numerous technology options to produce, transform, and
-provide energy services as well as to produce agriculture and forest
-products, and to determine land use and land cover. Using a run period
-extending from 1990 – 2100 at 5 year intervals, GCAM has been used to
-explore the potential role of emerging energy supply technologies and
-the greenhouse gas consequences of specific policy measures or energy
-technology adoption including; CO2 capture and storage, bioenergy,
-hydrogen systems, nuclear energy, renewable energy technology, and
-energy use technology in buildings, industry and the transportation
-sectors. GCAM is an Representative Concentration Pathway (RCP)-class
-model. This means it can be used to simulate scenarios, policies, and
-emission targets from various sources including the Intergovernmental
-Panel on Climate Change (IPCC). Output includes projections of future
-energy supply and demand and the resulting greenhouse gas emissions,
-radiative forcing and climate effects of 16 greenhouse gases, aerosols
-and short-lived species at 0.5×0.5 degree resolution, contingent on
-assumptions about future population, economy, technology, and climate
-mitigation policy.
-
-## Documentation
+## GCAM Model Overview
+
+### Introduction
+
+GCAM is a global integrated assessment model that represents the
+behavior of, and complex interactions between five systems: the energy
+system, water, agriculture and land use, the economy, and the
```

Comment:
Should we remove "water" from this line since it isn't in GCAM4.3?

Reply:
This won't be merged until the GCAM 4.4 release, if that makes any difference.

Reply:
Don't think so. Water will be in GCAM5 (I think)